frontend/src/components/explorer/gemBrowser/Subsystem.vue

```diff
@@ -105,8 +110,13 @@ export default {
       return {};
     }
 
+    const [compartments, compartmentLabel] = combineWords({
+      items: this.info.compartments.map(c => c.name),
+      itemType: 'compartment',
+    });
+
     const title = `${this.info.name}, Subsystem in ${this.model.short_name}`;
-    const description = `The subsystem ${this.info.name} in ${this.model.short_name} (version ${this.model.version}) can be found in ${this.info.compartments.length} compartments, and contains ${this.metabolites.length} metabolites and ${this.genes.length} genes.`;
+    const description = `The subsystem ${this.info.name} in ${this.model.short_name} (version ${this.model.version}) can be found in the ${compartments} ${compartmentLabel}, and contains ${this.metabolites.length} metabolites and ${this.genes.length} genes.`;
```

Comment:
We are only listing the amount of metabolites and genes in the meta data but at the interface, e.g. at `/explore/Human-GEM/gem-browser/subsystem/beta_alanine_metabolism`, you can see that there are also 17 reactions. Shouldn't we show that information as well?

Reply:
We could list these too and I could add a commit if we want this info. I was worried that it would generate descriptions that are too long since some subsystems have many more components. For reactions specifically we rely on a potentially slow request and only return max 1000 so it could be incomplete.

Reply:
Since these meta data is not supposed to be used by actual users, I think it makes sense to show only the count for genes, reactions and metabolites since otherwise the text can be rather tedious. 

However, for the compartments, I guess we can show all of them since there are not so many.